joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -69,11 +69,11 @@ class TestTrajectoryController : public ::testing::Test
 
   /// Publish trajectory msgs with multiple points
   /**
-   *  time_from_start - delay between each points
+   *  delay - delay between each points
```

Comment:
either correct the docstring or the variable name, but they should match

Reply:
fixed e68ead9